met-web/src/components/engagement/view/PreviewBanner.tsx

```diff
@@ -55,15 +60,43 @@ export const PreviewBanner = () => {
             }}
         >
             <Grid container direction="row" justifyContent="flex-end" alignItems="flex-start" padding={4}>
-                <Grid item xs={12}>
-                    <Typography variant="h4">Preview Engagement{!isDraft && ' - Published'}</Typography>
+                <Grid item container xs={12}>
+                    <Typography variant="h4" sx={{ mb: 2 }}>
+                        Preview Engagement{!isDraft && ' - Published'}
+                    </Typography>
                     <ConditionalComponent condition={isDraft}>
+                        <ConditionalComponent condition={imageExists === ''}>
+                            <Grid container alignItems="center" item xs={12}>
+                                <IconButton
+                                    color="info"
+                                    onClick={() => navigate(`/engagement/form/${engagementId}`)}
+                                    aria-label="no image"
+                                >
+                                    <ImageIcon />
+                                </IconButton>
+                                <Typography variant="body2">This engagement is missing a header image.</Typography>
+                            </Grid>
+                        </ConditionalComponent>
                         <ConditionalComponent condition={savedEngagement.surveys.length === 0}>
-                            <Typography variant="body2">This engagement is still missing a survey.</Typography>
+                            <Grid container alignItems="center" item xs={12}>
+                                <IconButton
+                                    color="info"
+                                    onClick={() => navigate(`/survey/create?engagementId=${engagementId}`)}
+                                    aria-label="no survey"
+                                >
+                                    <PollIcon />
+                                </IconButton>
+                                <Typography variant="body2">This engagement is missing a survey.</Typography>
+                            </Grid>
                         </ConditionalComponent>
-                        <Typography variant="body2" minHeight={25}>
-                            Please publish the engagement when ready.
-                        </Typography>
+                        <Grid container alignItems="center" item xs={12}>
+                            <IconButton color="info" aria-label="not published">
+                                <UnpublishedIcon />
+                            </IconButton>
+                            <Typography variant="body2" minHeight={25}>
```

Comment:
This typography has minHeight 25? Can you please show a screenshot of this?

Reply:
![image](https://user-images.githubusercontent.com/103138766/184737341-722ae3ed-4fba-48d7-a4dc-01bd95d2f38c.png)

That was already there before. Here is an image of the code.

Reply:
you can do above in a separate PR if it's not straightforward